R/codelist.R

```diff
@@ -50,12 +50,18 @@
 #'   \item p4.name: Polity IV country name
 #'   \item region: 7 Regions as defined in the World Bank Development Indicators
 #'   \item region23: 23 Regions as used to be in the World Bank Development Indicators (legacy)
+#'   \item un.intermediateregion.name: United Nations intermediate region name
+#'   \item un.intermediateregion.name: United Nations intermediate region code
```

Comment:
probably should be `\item un.intermediateregion.code: United Nations intermediate region code`

Reply:
Oops